service/frontend/service.go

```diff
@@ -343,12 +342,14 @@ func namespaceRPS(
 	frontendResolver membership.ServiceResolver,
 	namespace string,
 ) float64 {
-	hostRPS := float64(perInstanceRPSFn(namespace))
 	globalRPS := float64(globalRPSFn(namespace))
-	hosts := float64(numFrontendHosts(frontendResolver))
+	if globalRPS > 0 && frontendResolver != nil {
+		hosts := float64(numFrontendHosts(frontendResolver))
+		return globalRPS / hosts
+	}
 
-	rps := hostRPS + globalRPS*math.Exp((1.0-hosts)/8.0)
-	return rps
+	hostRPS := float64(perInstanceRPSFn(namespace))
```

Comment:
Shall we pick the lower value of hostRPS and globalRPS/#host?
If not, add a comment for globalRPS that it will override host rps in constants.go?